HdmiCec_2/HdmiCec_2.cpp

```diff
@@ -330,7 +330,8 @@ namespace WPEFramework
            HdmiCec_2::_instance = this;
            smConnection = NULL;
            InitializeIARM();
```

Comment:
Pls move the InitializeIARM and other HAL initialization to the Initialize() API and return empty string to indicate that it's initiated correctly. Any initialization in Initialize() should be cleaned up in DeInitialize() - https://github.com/rdkcentral/Thunder/blob/master/Source/plugins/IPlugin.h#L70